test_holiday_functions/test_purple_holiday_functions.py

```diff
@@ -53,8 +53,8 @@
 
 
 # Laurie
-
-
+assert naughty_nice(6,0)
+assert not home_for_the_holidays("John")
```

Comment:
Oops, you for the `_or_` in `naughty_or_nice`? 

Also, it's not required but it's standard convention in Python to add a space after the comma: 

```suggestion
assert naughty_or_nice(6, 0)
assert not home_for_the_holidays("John")
```